gitpod-io/gitpod
# Add dropdown shadow

## Description

This will add some elevation (shadow) to the dropdowns to indicate depth and content boundaries.

See [relevant discussion](https://gitpod.slack.com/archives/C01KPEPNBD0/p1684370565237249) (internal). Cc @axonasif 

| BEFORE | AFTER |
|-|-|
| <img width="400" alt="Frame 1514" src="https://github.com/gitpod-io/gitpod/assets/120486/7739a08e-293f-4e2b-b8df-6a93d681f1be"> | <img width="400" alt="Frame 1518" src="https://github.com/gitpod-io/gitpod/assets/120486/6cb5005a-0ff9-4861-a636-9f26737f207c"> |
| <img width="400" alt="Frame 1516" src="https://github.com/gitpod-io/gitpod/assets/120486/aa371019-0672-4df1-84a3-3131a5ebab5f"> | <img width="400" alt="Frame 1517" src="https://github.com/gitpod-io/gitpod/assets/120486/8c3126b5-9b85-4af8-b996-ba0e3e04bbda"> |

## How to test

Open some dropdowns (org selector, user menu, more actions, etc,.) and notice the shadow. 😎 

#### Preview status

gitpod:summary

## Build Options:

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold
